src/azure-cli/azure/cli/command_modules/dla/commands.py

```diff
@@ -187,5 +187,5 @@ def load_command_table(self, _):
         g.show_command('show', 'get')
         g.command('delete', 'delete')
 
-    with self.command_group('dla', is_preview=True):
+    with self.command_group('dla', deprecate_info=self.deprecate(expiration='2.67.0')):
```

Comment:
It's by design because I won't want to miss next breaking change window.

And as you may noticed, I didn't leverage the upcoming breaking change framework https://github.com/Azure/azure-cli/blob/b3199bd5af097c4fc32f7edbaca9a51c5ebf94c8/src/azure-cli-core/azure/cli/core/breaking_change.py#L584-L586

And @ReaNAiveD said that I will lose version info if I use default old `deprecated_info` rather than `register_command_group_deprecate`. That's another reason I explicitly declare `expiration` here